common/config/config.go

```diff
@@ -9,7 +9,7 @@ type Config struct {
 
 	APIServer struct {
 		Port         int    `envconfig:"STARHUB_SERVER_SERVER_PORT" default:"8080"`
-		ExternalHost string `envconfig:"STARHUB_SERVER_SERVER_EXTERNAL_HOST" default:"http://localhost"`
+		ExternalHost string `envconfig:"STARHUB_SERVER_SERVER_EXTERNAL_HOST" default:"localhost:8080"`
```

Comment:
suggest rename to `STARHUB_SERVER_PUBLIC_DOMAIN`

Reply:
Nice catch!  Renamed.